home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/cooperations/QueryCooperationIT.java

```diff
@@ -145,7 +145,7 @@ void invalidPageSizeTest() {
                 .build()
                 .perform())
             .matches((actual) -> actual.getCode() == 400)
-            .withMessageContaining("Parameter `arg1` is invalid - must be greater than or equal to 1.");
+            .withMessageContaining("Parameter `page_size` is invalid - must be greater than or equal to 1.");
     }
```

Comment:
The test passes, but the message "Parameter `arg2` is invalid - must be greater than or equal to 1." is not replaced by the message "Parameter `page_size` is invalid - must be greater than or equal to 1.", but they are combined into one, i.e. the following message comes in: "Parameter `arg2` is invalid - must be greater than or equal to 1. Parameter `page_size` is invalid - must be greater than or equal to 1.". It is need to think about this.

Reply:
I agree. Need to create a new Issue. And similarly solve the problem.

Reply:
The problem has been resolved.